documentation/tut/forecasting_scheduling.rst

```diff
@@ -49,27 +49,39 @@ How forecasting jobs are queued
 A forecasting job is an order to create forecasts based on measurements.
 A job can be about forecasting one point in time or about forecasting a range of points.
 
-
-In FlexMeasures, forecasting jobs are created by the server when new power, weather or price data arrives through the API (see :ref:`tut_posting_data`).
+In FlexMeasures, the usual way of creating forecasting jobs would be right in the moment when new power, weather or price data arrives through the API (see :ref:`tut_posting_data`).
 So technically, you don't have to do anything to keep fresh forecasts.
 
 The decision which horizons to forecast is currently also taken by FlexMeasures. For power data, FlexMeasures makes this decision depending on the asset resolution. For instance, a resolution of 15 minutes leads to forecast horizons of 1, 6, 24 and 48 hours. For price data, FlexMeasures chooses to forecast prices forward 24 and 48 hours
 These are decent defaults, and fixing them has the advantage that scheduling scripts (see below) will know what to expect. However, horizons will probably become more configurable in the near future of FlexMeasures. 
 
+You can also add forecasting directly jobs via the CLI. We explain this practice in the next section. 
+
+
+
 Historical forecasts
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 There might be reasons to add forecasts of past time ranges. For instance, for visualisation of past system behaviour and to check how well the forecasting models have been doing on a longer stretch of data.
 
 If you host FlexMeasures yourself, we provide a CLI task for adding forecasts for whole historic periods. This is an example call:
 
+Here we request 6-hour forecasts for two sensors, for a period of two days:
+
 .. code-block:: console
 
-     flexmeasures add forecasts --from-date 2020-01-02 --to-date 2020-6-30 --horizon_hours 6  --resolution 60 --asset-id 2
+    flexmeasures add forecasts --sensor-id 2 --sensor-id 3 \
```

Comment:
This command doesn't return any indication of success. I was expecting a:
`New forecasting job has been added to the queue.`